flutter_vlc_player/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.1.0
```

Comment:
If we do breaking changes - it looks like it should be 7.0.0 then
https://semver.org/spec/v2.0.0-rc.1.html#spec-item-9

Reply:
Only the following classes are changed `HwAcc` and `VlcPlayerState`. 
The second one is not used anywhere, i guess we should remove it from code. 
Making a major revision only for a small change doesn't seem reasonable to me.

Reply:
Let's remove `VlcPlayerState` then and as for HwAcc - it looks like a breaking change. I think our promise is to make breaking changes only with major version updates. While it's a small change it could break our users' code and could put a lot of frustration.